earthfile2llb/converter.go

```diff
@@ -78,12 +79,16 @@ type Converter struct {
 	directDeps          []*states.SingleTarget
 	buildContextFactory llbfactory.Factory
 	cacheContext        pllb.State
+	runOpts             []llb.RunOption
+	persistentCacheDirs map[string]struct{} // Treated as a "set" of cache directories to persist in the image
```

Comment:
nit: comment is no longer needed as `map[string]string` was changed to `map[string]struct{}`.
```suggestion
	persistentCacheDirs map[string]struct{}
```